.github/workflows/ci.yml

```diff
@@ -15,7 +15,10 @@ jobs:
       matrix:
         # Test on the oldest supported, the latest stable, and nightly
         deno: [old, stable, canary]
-        os: [macOS-latest, windows-latest, ubuntu-22.04-xl]
+        os: [macOS-latest, windows-latest, ubuntu-latest]
+
+    env:
+      DENO_FUTURE: ${{ matrix.deno == 'old' && '1' || '0' }}
```

Comment:
This is needed to enable support for lockfile v4.

Reply:
It's a bit weird to test with DENO_FUTURE=1, it defeats the purpose of testing with 1.46

Reply:
That kinda makes sense - Would you agree to bump the minimum required version to 2.0?

Reply:
Maybe we can test with --no-lock when testing with 1.X?

Reply:
I tried that now, but turned out it's actually involved, because test cases in [help_test.ts](https://github.com/denoland/deployctl/blob/8644ec52d28f9f9c57d30535f4a1a2a93cc9f093/tests/help_test.ts) spawn another Deno process to run `deployctl.ts` and we need to pass `--no-lock` to it but `Deno.args` is an empty array when running `deno test`.

Reply:
I did a hacky approach in [3e0d9a0](https://github.com/denoland/deployctl/pull/332/commits/3e0d9a08da1c5cb19bf7da030f1e9a2c8cf14c36) where `--no-lock` is appended based on the semver comparison.